docs/reference/configuration/magic_strings.rst

```diff
@@ -246,3 +247,23 @@ Experiment dependent variables
    ::
 
       FORWARD_MODEL JOB_NAME(<ARG0>="/your/run/path/realization-4/iter-1")
+
+
+.. _ms_ertcase:
+.. topic:: ERTCASE
+
+   Will be replaced with the current case name defined when starting the experiment.
+
+
+   *Example:*
+
+   ::
+
+      RUNPATH output/<ERTCASE>/realization-<IENS>/iter-<ITER>
+
+   In this example when running the experiment and given that the user does not change the `default` case name
```

Comment:
Change the default case name using keyword RUNPATH? If so, should we refer to that?

Reply:
Not sure I understand your comment. You cannot change the case name using the RUNPATH keyword. The case name is set from the GUI, by adding a case in the case selector dropdown.

Reply:
I thought you implied that the change was from the configuration, not during runtime. Otherwise it would have made sense to refer to features used to change the case name.

Disregard this then. :+1: